website/docs/extensions/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 sidebar_position: 110
-title: Writing extensions
+title: Extensions
 description: Writing or deploying extensions.
```

Comment:
description should be Developing instead of Writing ?

Reply:
Missed that! Thank you.